### ISSUE TITLE
Custom maximum scale support

### ISSUE THREAD

**Issue by duncanista** [semakers/zoom-widget#8]
I think it would be wonderful if you could set the **scale** as a custom property. I have been using other widgets to zoom, but yours is truly helpful, and I would love if this was added.

By doing some testing I have found that ```1.0``` is the maximum scale for every image, value given by the ```onScaleUpdate``` function. 

Hope it can be added, anyhow, will try to understand the code to see if I can do a PR.

**Comment by semakers** [comment]
yes, the maximum limit, the minimum limit and the use of a controller to inject the width, height and be able to manipulate some widget methods without affecting your child will be added

**Comment by duncanista** [comment]
If you need help, let me know.
I see you are from Puebla, so, encantado de ayudar a un paisano. :)
